docs/source/examplecleanup.rst

```diff
@@ -6,55 +6,106 @@ End-to-End Example
 
 
-#. Cleanup repo using ``BFG`` and the above passwords file
+#. Cleanup repo using ``BFG`` and the above remove.txt file
 
    There's a very slick tool designed to cleanup git commit history called ``BFG``: https://rtyley.github.io/bfg-repo-cleaner/. By default ``BFG`` doesn't modify the contents of your latest commit on your master (or 'HEAD') branch, even though it will clean all the commits before it. This of course means if you have active code with "bad passwords" tartufo will still fail, but let's take the bulk of the old entries out first.
 
    .. code-block:: console
 
-      # Cleanup with
+      # Cleanup with BFG
       wget https://repo1.maven.org/maven2/com/madgag/bfg/1.13.0/bfg-1.13.0.jar
-      git clone --mirror git@${GITHUBADDRESS}:${GITHUBPROJECT}/${GITHUBREPO}
       # Make a backup
       cp -r ${GITHUBREPO} backup_${GITHUBREPO}
-      java -jar bfg-1.13.0.jar --replace-text badpwz.txt ${GITHUBREPO}
+      java -jar bfg-1.13.0.jar --replace-text remove.txt ${GITHUBREPO}
+
+
+#. Uh Oh!
+
+   Occasionally the results will be to big to process all at once. If that happens simply split up the results and loop through them.
+
+   .. code-block:: console
+
+      # occasionally the results will be to big to process all at once
+      split -l 200 remove.txt
+      for f in x*; do java -jar bfg-1.13.0.jar --replace-text $f ${GITHUBREPO}; done
+
+
+#. Proceed with cleanup/audit
+
+   Now you have removed the low hanging fruit, it's time to look at the tough stuff
+
+   .. code-block:: console
+
+      #run tartufo again to check for icky code straglers in active code
+      leftovers=`tartufo --regex --no-cleanup ${GITHUBREPO}`
+      tmppath=`echo -e "$leftovers" | tail -n1 | awk '{print $6}'`
+      #look through the remaining strings, if there's anything that looks like it shouldn't be there, dig into it and clear it out
+      cat ${tmppath}/* | jq '. | " \(.file_path) \(.matched_string) \(.signature)"' | sort -u
+
+
+#. Take a good look at the output of the above, make sure there are no secrets or other sensitive data remaining.
+
+   Now you are going to exclude the signatures for the remaining items (which you have verified are non-risk)
+
+   .. code-block:: console
+
+      # now you're ready to ignore those webhook urls:
+      cat ${tmppath}/* | jq -r '.signature' | sort -u > allsignatures.txt 
+      sed -i -e 's/$/\",/g' -e 's/^/  \"/g' allsignatures.txt
+      linestr=`grep -n 'exclude-signatures = \[' pyproject.toml`
+      line=`echo $linestr | cut -d ":" -f 1`
+      line=$(($line+1))
+      { head -n $(($line-1)) pyproject.toml; cat allsignatures.txt; tail -n +$line pyproject.toml; } > pyproject.toml_new
+      mv pyproject.toml pyproject.toml_bak
+      mv pyproject.toml_new pyproject.toml
+      # one final run to make sure your signatures are all set
+      poetry run tartufo ${gitrepo} --regex --no-cleanup
```

Comment:
I think this underscores the need for a new command like `tartufo creation-exclusion-list` once #78 is done and merged. 😄

Reply:
yeah, that would be pretty sweet!